kci_build

```diff
@@ -196,6 +196,9 @@ def _show_build_env(build_env, arch=None):
         xc = build_env.get_cross_compile(args.arch)
         if xc:
             print(xc)
+        xc_compat = build_env.get_cross_compile_compat(args.arch)
```

Comment:
The only issue here is that if for any reason `cross_compile_compat` is define but not `cross_compile`, then the compat value will be printed on the line where one would normally expect to see the cross compiler.  This probably won't be a problem in practice as I guess you need to be cross-compiling first if you also want to have a compat toolchain, but there's nothing in the configuration to enforce it.

So it's probably better to print an empty string when a value is not there to always have the same number of lines being output.  Ideally these should be key-value pairs but then it makes it a bit harder to parse...